diff -Nru a/src/lsmmathmlmathelement.c b/src/lsmmathmlmathelement.c
--- a/src/lsmmathmlmathelement.c	2024-03-01 21:34:13.302151905 -0800
+++ b/src/lsmmathmlmathelement.c	2024-03-01 21:34:24.598431788 -0800
@@ -146,7 +146,7 @@
 	style->very_thick_math_space = 		LSM_MATHML_SPACE_EM_VERY_THICK * style->math_size;
 	style->very_very_thick_math_space = 	LSM_MATHML_SPACE_EM_VERY_VERY_THICK * style->math_size;
 
-	style->math_family = g_strdup ("Serif");
+	style->math_family = g_strdup ("KaTeX_Main");
 	style->math_variant = LSM_MATHML_VARIANT_NORMAL;
 	style->math_color.red = 0;
 	style->math_color.green = 0;
diff -Nru a/src/lsmmathmlmathelement.h b/src/lsmmathmlmathelement.h
--- a/src/lsmmathmlmathelement.h	2024-03-01 21:34:13.302151905 -0800
+++ b/src/lsmmathmlmathelement.h	2024-03-01 21:34:24.598431788 -0800
@@ -29,10 +29,10 @@
 
 G_BEGIN_DECLS
 
-#define LSM_MATHML_FONT_SERIF		"Serif"
-#define LSM_MATHML_FONT_SANS		"Sans"
-#define LSM_MATHML_FONT_MONOSPACE	"Monospace"
-#define LSM_MATHML_FONT_FRAKTUR		"Serif"
+#define LSM_MATHML_FONT_SERIF		"KaTeX_Main"
+#define LSM_MATHML_FONT_SANS		"KaTeX_SansSerif"
+#define LSM_MATHML_FONT_MONOSPACE	"KaTeX_Typewriter"
+#define LSM_MATHML_FONT_FRAKTUR		"KaTeX_Fraktur"
 #define LSM_MATHML_FONT_SCRIPT		"cmsy10"
 #define LSM_MATHML_FONT_DOUBLE_STRUCK	"msbm10"
 
diff -Nru a/src/lsmmathmlpresentationtoken.c b/src/lsmmathmlpresentationtoken.c
--- a/src/lsmmathmlpresentationtoken.c	2024-03-01 21:34:13.302151905 -0800
+++ b/src/lsmmathmlpresentationtoken.c	2024-03-01 21:34:24.598431788 -0800
@@ -96,7 +96,8 @@
 	LsmMathmlPresentationToken *token = LSM_MATHML_PRESENTATION_TOKEN (self);
 	LsmMathmlLength length;
 
-	if (token->type == LSM_MATHML_PRESENTATION_TOKEN_TYPE_IDENTIFIER) {
+	if (token->type == LSM_MATHML_PRESENTATION_TOKEN_TYPE_IDENTIFIER &&
+	    style->math_variant != LSM_MATHML_VARIANT_MONOSPACE) {
 		char *text;
 		text = lsm_mathml_presentation_token_get_text (token);
 		style->math_variant = g_utf8_strlen (text, -1) > 1 ?
diff -Nru a/src/lsmmathmlview.c b/src/lsmmathmlview.c
--- a/src/lsmmathmlview.c	2024-03-01 21:34:13.302151905 -0800
+++ b/src/lsmmathmlview.c	2024-03-01 21:34:24.598431788 -0800
@@ -44,7 +44,7 @@
 
 static const char *lsm_mathml_font_names[] = {
 	"",
-	"Sans",
+	"KaTeX_SansSerif",
 	"cmr10",
 	"cmmi10",
 	"cmex10",
diff -Nru a/src/lsmsvgstyle.c b/src/lsmsvgstyle.c
--- a/src/lsmsvgstyle.c	2024-03-01 21:34:13.306152005 -0800
+++ b/src/lsmsvgstyle.c	2024-03-01 21:34:24.598431788 -0800
@@ -232,7 +232,7 @@
 		.name = "font-family",
 		.id = LSM_PROPERTY_OFFSET_TO_ID (LsmSvgStyle, font_family),
 		.trait_class = &lsm_null_trait_class,
-		.trait_default = "Sans" /* Depends on user agent */
+		.trait_default = "KaTeX_SansSerif" /* Depends on user agent */
 	},
 	{
 		.name = "font-size",
